.github/workflows/deploy.yml

```
@@ -0,0 +1,246 @@
+                    "elements": [
+                      {
+                        "type": "mrkdwn",
+                        "text": "<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|배포 상세 정보 보기>"
+                      }
+                    ]
+                  }
+                ]
+              }
+            ]
+          }
+EOF
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 241: could not find expected ':'

(syntax-check)